mkdocs/docs/tutorials/deployment-patterns/fixed-split.md

```diff
@@ -0,0 +1,325 @@
+---
+template: main.html
+---
+
+# Fixed Split Deployment
+
+!!! tip "Scenario: FixedSplit deployment"
+
+    [FixedSplit deployment](../../../concepts/buildingblocks/#deployment-patterns), as the name indicates, is meant for scenarios where you do **not** want Iter8 to shift traffic between versions during the experiment. In this tutorial, you will:
+
+    * Modify the [quick start tutorial](../../../getting-started/quick-start) to use FixedSplit instead of Progressive deployment.
+
+    The modified A/B testing experiment with FixedSplit deployment pattern is depicted below.
+
+    ![Canary](../../images/fixedsplitab-exp.png)
+
+
+???+ warning "Before you begin, you will need... "
+    > **Note:** Please choose the same K8s stack (for example, Istio, KFServing, or Knative) consistently throughout this tutorial. If you wish to switch K8s stacks between tutorials, start from a clean K8s cluster, so that your cluster is correctly setup.
+
+## Steps 1 to 3
+    
+Please follow steps 1 through 3 of the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster).
+
+## 4. Create versions of your application
```

Comment:
As I read through this tutorial, I did not understand where the traffic split between versions was specified or set up. I do see it when I examine the yaml files. But I think we should be explicit that the split is set up during the install of the app/model. I don't know if this should be in this section or as part of the scenario description or both.